examples/systemd/rest-server.service

```diff
@@ -10,13 +10,35 @@ Group=www-data
 ExecStart=/usr/local/bin/rest-server --path /path/to/backups
 Restart=always
 RestartSec=5
+# Makes created files group-readable, but inaccessible by others
+UMask=027
 
-# Optional security enhancements
+# Recommended security enhancements using features present in systemd version 247
+CapabilityBoundingSet=
+LockPersonality=true
+MemoryDenyWriteExecute=true
 NoNewPrivileges=yes
 PrivateTmp=yes
+PrivateDevices=true
+PrivateUsers=true
 ProtectSystem=strict
 ProtectHome=yes
+ProtectClock=true
+ProtectControlGroups=true
+ProtectKernelLogs=true
+ProtectKernelModules=true
+ProtectKernelTunables=true
+ProtectProc=invisible
+ProtectHostname=true
+ProcSubset=pid
 ReadWritePaths=/path/to/backups
```

Comment:
Please move that line somewhere near `ExecStart`. Spotting that this line is the only one that must be changed within a block of 25 options could be a bit frustrating.